src/server/routes/routes.ts

```diff
@@ -135,7 +135,8 @@ router.post(
           }
         });
       });
-    } catch (error: unknown) {
+    } catch (error: any) {
+      console.log(error);
```

Comment:
If we want this error to be logged, use the logger that we have setup for middleware.

Reply:
@Hyperkid123 Not sure if I did correctly. Let me know if this is wrong.

Reply:
Yup